src/scss/_custom.scss

```diff
@@ -1,3 +1,104 @@
-p {
-  font-size: 30pt;
+body {
+  margin: 0;
+  background: #ffffff;
+  color: #000000;
+}
+
+.page-content {
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 80%;
+  margin-bottom: 5px;
+}
+
+.user-sidebar {
+  display: block;
+  margin: auto;
+  margin: auto;
+  width: 26%;
+  height: 600px;
+  float: left;
+  border-bottom: 1px solid #A1A1A1;
+  // background: #B4DBD7;
+
+      img {
+        border-radius: 8px;
+        margin-right: auto;
+        margin-left: auto;
+        width: 99%;
+        margin-bottom: 10px;
+      }
+
+} //END USER-SIDEBAR ICON
+
+
+.name-login {
+  margin-top: 10px;
+  text-align: left;
+  line-height: 1px;
+  display: block;
+  margin-bottom: 10px;
+  border-bottom: 1px solid #A1A1A1;
+
+        h3 {
+          font-size: 20pt;
+          color: #000000;
+          font-family: Arial, sans-serif;
+          font-style: bold;
+          margin-top: 10px;
+        }
+
+        h4 {
+          font-size: 13pt;
+          color: #000000;
+          margin-bottom: 10px;
+        }
+
+} //END NAME-LOGIN
+
+
+.details {
+  display: block;
+  border-bottom: 1px solid #A1A1A1;
+  line-height: .2px;
+
+    p {
+      color: #000000;
+      font-size: 10pt;
+    }
+
+    span {
+      font-size: 13px;
+      color: #A1A1A1;
+    }
+}
+
+
+table {
+  margin-top: 15px;
+
+    td {
+      text-align: center;
+      margin-right: auto;
+      margin-left: auto;
+      width: 15%;
+      // padding-right: 30px;
+      padding: 10px;
+      line-height: .5px;
+    }
+
+}
```

Comment:
Please make me a promise not to use `<table>` for a three-column layout... What about that grid?